app/controllers/codelists-management/codelist.ts

```diff
@@ -16,7 +17,9 @@ export default class CodelistController extends Controller {
     event.preventDefault();
 
     await Promise.all(
-      this.model.codelist.concepts.map((option) => option.destroyRecord()),
+      this.model.codelist.concepts
+        .filter((model) => !(model instanceof IconModel))
+        .map((option) => option.destroyRecord()),
```

Comment:
**Note:** Prevent destroy when it's an icon. Icons can be destroyed in the icon-catalog.